app/src/main/java/io/intrepid/contest/models/Contest.java

```diff
@@ -134,5 +110,13 @@ public Contest build() {
             contest.categories = this.categories;
             return contest;
         }
+
+        public List<Category> getCategories() {
+            return categories;
+        }
+
+        public void setCategories(List<Category> categories) {
```

Comment:
I think setCategories should be part of the builder class so we want this to return builder.